compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -261,6 +261,13 @@ OMR::ARM64::TreeEvaluator::aloadEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    TR::MemoryReference *tempMR = new (cg->trHeapMemory()) TR::MemoryReference(node, 8, cg);
    generateTrg1MemInstruction(cg, TR::InstOpCode::ldrimmx, node, tempReg, tempMR);
 
+#ifdef J9_PROJECT_SPECIFIC
```

Comment:
We should not be introducing `J9_PROJECT_SPECIFIC` macros into OMR.  TreeEvaluator is an extensible class so there is a mechanism in place for introducing project specialization.  I suggest either coming up with some generic, post-load function that can be implemented in OMR (that does nothing) and performs the masking in an OpenJ9 specialization, or introducing a base `generateVFTMaskInstruction` in OMR (that does nothing) such that this code doesn't need to be guarded.

Reply:
Removed the `J9_PROJECT_SPECIFIC` guard based on the review comment.